OpenRA.Mods.RA/Traits/Infiltration/Infiltrates.cs

```diff
@@ -58,6 +58,10 @@ bool IsValidOrder(Actor self, Order order)
 			if (order.ExtraData == 0 && order.TargetActor == null)
 				return false;
 
+			// Should not infiltrate own or allied
+			if (order.TargetActor.Owner.IsAlliedWith(self.Owner))
```

Comment:
This was reported as a bug. This was considered a bug even in RA2. I still don't see the gain of putting this under a boolean of say, `AllowInfiltrateAllies`.